src/lib/tabs/tab-group.ts

```diff
@@ -168,18 +168,27 @@ export class MatTabGroup extends _MatTabGroupMixinBase implements AfterContentIn
 
     // If there is a change in selected index, emit a change event. Should not trigger if
     // the selected index has not yet been initialized.
-    if (this._selectedIndex != indexToSelect && this._selectedIndex != null) {
-      const tabChangeEvent = this._createChangeEvent(indexToSelect);
-      this.selectedTabChange.emit(tabChangeEvent);
-      // Emitting this value after change detection has run
-      // since the checked content may contain this variable'
-      Promise.resolve().then(() => this.selectedIndexChange.emit(indexToSelect));
+    if (this._selectedIndex != indexToSelect) {
```

Comment:
Can we use `!==` here?

Reply:
That's what was being used here before so I kept it. I'm guessing it was there to handle somebody passing the index in as a string.